src/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import {
+  AnimatedPlaceholderProps,
+  InputProps,
+  InputWrapperProps,
+  StyledIconWrapperProps,
+  TextAreaProps,
+  WrapperProps,
+} from "./type";
+
+const Wrapper: React.FC<WrapperProps> = ({
```

Comment:
Move it to other atoms and make it more generic.
we have card atom.
modify it or create a generic atoms so that it can be utilized by other atoms also

Reply:
no need